app/ledger.js

```diff
@@ -458,17 +458,19 @@ var initialize = (paymentsEnabled, reason) => {
           // Scenario: User enables Payments, disables it, waits 30+ days, then
           // enables it again -> reconcileStamp is in the past.
           // In this case reset reconcileStamp to the future.
-          if (reason === 'changeSettingPaymentsEnabled') {
```

Comment:
as a result of this PR we can remove `changeSettingPaymentsEnabled` at ln 134 and the `reason` arg in this function `initialize(paymentsEnabled,reason)` (it was added to fix this)